apps/docs/pages/guides/auth/sessions.mdx

```diff
@@ -0,0 +1,151 @@
+
+This approach has some significant technical drawbacks, as it meant that the authentication server and database was central and inseparable from the entirety of the application:
+
+- Requirement to be always available and performant. If the authentication server or its database crashed or wasn't available for even a few seconds, the whole application went down. Scheduling maintenance or dealing with transient errors often became impossible to manage.
+- A failing authentication server could cause a chain of failures across other systems and APIs, paralyzing the whole application system.
+- It is generally very hard to globally distribute workloads for authentication, so this causes a situation where all requests experience the latency to communicate with this primary region.
+
+Access and refresh tokens are therefore a better approach because the session information is encoded in a short-lived token -- the access token -- which can be transferred across APIs and systems without relying on the availability or performance of a central server. An application can thus tolerate transient failures or performance issues a lot better. By trying to refresh the access token ahead of time, your application can safely continue to function even with larger outages.
+
+It's better for cost optimization and scaling as well, as the authentication system's servers and database only handle traffic for this use case.
+
+### How do I make sure that an access token (JWT) cannot be used after a user clicks sign out?
+
+Most applications rarely need such strong guarantees. Consider adjusting the JWT expiry time to an acceptable value. If this is still necessary, you should try to use this validation logic only for the most sensitive actions within your application.
+
+When a user clicks sign out, the sessions affected by the logout are removed from the database entirely. You can check that the `session_id` claim in the JWT corresponds to a row in the `auth.sessions` table. If such a row does not exist, it means that the user has logged out.
+
+Note that sessions are not proactively terminated when their maximum lifetime (time-box) or inactivity timeout are reached. These sessions are cleaned up progressively 24 hours after reaching that status. This allows you to tweak the values or roll back changes without causing unintended user friction.
+
+### Can I use HTTP-Only cookies to store the access and refresh tokens?
+
+This is possible, but only for apps that use the traditional server-only web app approach where all of the application logic is implemented on the server and it returns rendered HTML only.
+
+If your app uses any client side JavaScript to build a rich user experience, using HTTP-Only cookies is not possible since only your server will be able to see and refresh the session of the user; while the browser would not have access to the access and refresh token.
```

Comment:
```suggestion
If your app uses any client side JavaScript to build a rich user experience, using HTTP-Only cookies is not possible since only your server will be able to see and refresh the session of the user. The browser would not have access to the access and refresh token.
```